packages/@aws-cdk/aws-cognito/lib/user-pool-client.ts

```diff
@@ -451,7 +451,7 @@ export class UserPoolClient extends Resource implements IUserPoolClient {
         'DescribeCognitoUserPoolClient',
         {
           resourceType: 'Custom::DescribeCognitoUserPoolClient',
-          onCreate: {
+          onUpdate: {
```

Comment:
Should `onCreate` be completely removed?

Reply:
Yep! It seems to be relatively unknown that `onCreate` defaults to the value of on `onUpdate` when not specified (see `@default` for https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.custom_resources.AwsCustomResource.html#oncreate).

Please check me on this and confirm that the changes to integration test snapshots _add_ `Update` but _do not_ remove `Create`?

https://github.com/aws/aws-cdk/blob/4c62f0b31911ededb5542030ee03fec17b3af994/packages/%40aws-cdk/custom-resources/lib/aws-custom-resource/aws-custom-resource.ts#L435-L446

I think this is actually a common bug (specifying `onCreate` but not `onUpdate` for `AwsCustomResource`), even within `@aws-cdk/*`.

Reply:
I did not know that, thanks for the clarification! 

Yep, `OnCreate` is still in the snapshots